### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Update Backend](https://trello.com/c/WrWw93Q9/752-update-backend)

File: datatypes/src/util/identifiers.rs [hunk 1]
```diff
@@ -4,6 +4,7 @@ pub trait Identifier: Sized {
 }
 
 #[macro_export]
+#[allow(clippy::crate_in_macro_def)]
```
Comment (ChristianBeilschmidt) [inline datatypes/src/util/identifiers.rs:7]:
```suggestion
```
Why don't call it `$crate` like at the other places?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: datatypes/src/util/identifiers.rs (hunk 1 above)
Comment (michaelmattig) [reply datatypes/src/util/identifiers.rs:7]: because it actually is `crate` and not `$crate` 🙈 maybe a bad smell but I don't know how to do it differently

Comment (ChristianBeilschmidt) [reply datatypes/src/util/identifiers.rs]: [f3750c1](https://github.com/geo-engine/geoengine/pull/524/commits/f3750c15db460ba30bf9831ede6d603cf6990158)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**